ledger-service/http-json-perf/BUILD.bazel

```diff
@@ -103,19 +92,6 @@ daml_compile(
             "-Dlogback.configurationFile=$(location :release/json-api-perf-logback.xml)",
         ],
         main_class = "com.daml.http.perf.Main",
-        scala_deps = [
-            "@maven//:com_github_scopt_scopt",
-            "@maven//:com_typesafe_akka_akka_actor",
-            "@maven//:com_typesafe_akka_akka_http_core",
-            "@maven//:com_typesafe_akka_akka_stream",
-            "@maven//:com_typesafe_scala_logging_scala_logging",
-            "@maven//:org_scalactic_scalactic",
-            "@maven//:org_scalatest_scalatest_core",
-            "@maven//:org_scalatest_scalatest_matchers_core",
-            "@maven//:org_scalatest_scalatest_shouldmatchers",
-            "@maven//:org_scalatest_scalatest_wordspec",
-            "@maven//:org_scalaz_scalaz_core",
-        ],
```

Comment:
These were all dependencies from the `da_scala_library`, so no need to repeat them here.